cirq/ops/common_gates.py

```diff
@@ -122,23 +122,32 @@ def controlled(self,
                    control_qid_shape: Optional[Tuple[int, ...]] = None
                   ) -> raw_types.Gate:
         """
-        Constructs CXPowGate from controlled XPowGate when applicable.
-
-        This method is a specialized controlled method for XPowGate. It
-        overrides the default behavior of returning a ControlledGate by
-        transforming the underlying controlled gate to a CXPowGate and
-        removing the last specified control qubit (which acts first
-        semantically).  If this is a gate with multiple control qubits, it will
-        now be a ControlledGate with one less control.
-
-        This behavior only occurs when the last control qubit is a default-type
-        control qubit. A default-type control qubit is one with shape of 2 (not
-        a generic qudit) and where the control is satisfied by the qubit being
-        ON, as opposed to OFF. Note also that this only transforms into a
-        CXPowGate (or controlled version of that gate) if the global shift on
-        the XPowGate is 0, otherwise it produces a normal ControlledGate.
-
-        (Note that a CXPowGate is, by definition, a controlled-XPowGate.)
+        Returns a `CXPowGate` or a controlled `CXPowGate` when possible.
```

Comment:
```suggestion
        Returns a `CXPowGate` or a controlled `XPowGate` when possible.
```

Reply:
I think this (and others below) might be intentionally referring to cases where there are multiple controls which would result in a controlled `CXPowGate`. But I might be wrong. Would also like to know if there are other suggestions on the wording here.

Reply:
I think you're right, thanks. I'll remove my noisy comments :) 
I'm going to look at the actual code and try to understand what's happening.

Reply:
If I understand correctly the whole idea here is that we have named CX and CCX gates, let's convert them when we can. 

What do you think about this? A bit wordy.

```
Returns a Controlled XPowGate and, if possible, uses the named `CXPowGate` at the end of the control chain.
```

Reply:
How about
```
Returns a controlled `XPowGate`, using a `CXPowGate` where possible.
```
I think this is one of those cases where the top most string cannot carry the weight of the exceptions, but we want to warn the reader that they need to read more.  This accurately describes the result of the method, and say "but watchout!"  The watchout stuff is then described in the rest of the docstring.

Reply:
This version sounds good to me!